README.rst

```diff
@@ -21,27 +21,88 @@ The executable exposes the REST (or other) services from the different engines.
 .. image:: https://api.codacy.com/project/badge/Grade/272258ac93e847b9b61c08d4144d0538
     :target: https://www.codacy.com/app/woutslakhorst/nuts-go
 
-.. inclusion-marker-for-contribution
+Dependencies
+************
 
-Installation
-------------
+This projects is using go modules, so version > 1.12 is recommended. 1.10 would be a minimum.
```

Comment:
Since we use some error handling of 1.13, should this be 1.13?

Reply:
true